Tweak.x

```diff
@@ -1,10 +1,22 @@
 #import "../YouTubeHeader/YTVideoQualitySwitchOriginalController.h"
 
+BOOL makeItFake = NO;
+
+%hook YTVersionUtils
+
++ (NSString *)appVersion {
+    return makeItFake ? @"18.18.2" : %orig;
+}
+
+%end
+
 %hook YTVideoQualitySwitchControllerFactory
 
 - (id)videoQualitySwitchControllerWithParentResponder:(id)responder {
     Class originalClass = %c(YTVideoQualitySwitchOriginalController);
+    makeItFake = YES;
     return originalClass ? [[originalClass alloc] initWithParentResponder:responder] : %orig;
+    makeItFake = NO;
```

Comment:
This line is never executed. Your change causes YouTube version to permanently be `18.18.2` after `-[YTVideoQualitySwitchControllerFactory videoQualitySwitchControllerWithParentResponder:]` is called. Is this intentional?

Reply:
my bad. tried use timer to set it back to NO and guess that booleans in this case unnecessary since YTVideoQualitySwitchControllerFactory runs only once, so if i close player and open any video again it wont set makeItFake to YES and new selector will pop-up.

at the moment there are no global differences between 18.25.1 and 18.18.2 versions to keep older one permanently, but i have no doubt that some kind of error may occur with subsequent updates.